components/scanners/zaproxy/go.mod

```diff
@@ -6,10 +6,10 @@ require (
 	github.com/go-errors/errors v1.5.1
 	github.com/jonboulle/clockwork v0.5.0
 	github.com/smithy-security/pkg/env v0.0.1
-	github.com/smithy-security/pkg/sarif v0.0.4
-	github.com/smithy-security/smithy/sdk v0.0.11-alpha
+	github.com/smithy-security/pkg/sarif v0.0.5-0.20250429185943-362b30bf2d13
+	github.com/smithy-security/smithy/sdk v0.0.11-alpha.0.20250429114645-86a9857fcaf9
```

Comment:
If we made any substantial changes to the SDK, can you please bump the actual version?

Reply:
yes, please review

-  https://github.com/smithy-security/smithy/pull/894
- https://github.com/smithy-security/pkg/pull/11

then we can merge it and replace the hashes with a version

Reply:
fixed